cliquet/__init__.py

```diff
@@ -327,3 +340,16 @@ def initialize_cliquet(config, version=None, project_name=None):
     # Include cliquet views with the correct api version prefix.
     config.include("cliquet", route_prefix=api_version)
     config.route_prefix = api_version
+
+
+def wrap_app(app, settings):
+    "Wraps the passed wsgi application with wsgi middlewares."
+
+    # Setup new-relic.
+    if settings.get('cliquet.newrelic_config', False):
+        ini_file = settings.get('cliquet.newrelic_config')
+        env = settings.get('cliquet.newrelic_env', 'dev')
```

Comment:
Well, I believe we should keep up with gathering all default settings in one place. It's also a very easy way to get a list of all settings handled by cliquet :)